gbo/.htaccess

```diff
@@ -0,0 +1,13 @@
+Options +FollowSymLinks
+
+# Turn off MultiViews
+
+Options -MultiViews
+
+
+# Rewrite engine setup
+
+RewriteEngine On
+
+
+RewriteRule ^$ https://wawrzik.github.io/GENIALOntologies/index-en.html [R=301,NE,L]
```

Comment:
Be aware this is 301 Moved Permanently, so if you change the redirect in the future, that could cause issues.